fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/PrivateBrowsingDescriptionViewHolder.kt

```diff
@@ -121,3 +136,56 @@ private fun PrivateBrowsingDescriptionPreview() {
         )
     }
 }
+
+/**
+ * Total Private Browsing Mode description.
+ *
+ * @param onLearnMoreClick Invoked when the user clicks on the who can see my activity link.
+ */
+@Composable
+fun FeltPrivacyModeDescription(
+    onLearnMoreClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .wrapContentHeight(),
+        shape = RoundedCornerShape(8.dp),
+        backgroundColor = FirefoxTheme.colors.layerAccentNonOpaque,
+        elevation = 0.dp,
+    ) {
+        Column(
+            modifier = Modifier.padding(PaddingValues(16.dp)),
+        ) {
+            Text(
+                text = stringResource(id = R.string.felt_privacy_desc_card_title),
+                style = FirefoxTheme.typography.headline7,
+                color = FirefoxTheme.colors.textPrimary,
+                lineHeight = 20.sp,
+            )
+
+            Spacer(modifier = Modifier.height(8.dp))
+
+            ClickableSubstringLink(
```

Comment:
We're looking to deprecate the usage of this in favor of `LinkText` in https://github.com/mozilla-mobile/firefox-android/pull/3260.

Reply:
gotcha, I'll refactor this into a `LinkText`